src/char/int_party.c

```diff
@@ -614,6 +614,7 @@ int mapif_parse_PartyLeave(int fd, int party_id, int account_id, int char_id)
 {
 	struct party_data *p;
 	int i,j;
+	unsigned int leader;
```

Comment:
I've used unsigned integer because party structure uses it (at: p->party.member[i].leader;), it's safe to change to int as its values are only 0 or 1, but it will be different from the structure, so should I change?